src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -330,17 +331,25 @@ default PlayerLookupService playerLookupService() {
     }
 
     @ConfigItem(
-        keyName = "screenshotHideChat",
+        keyName = "chatPrivacy",
         name = "Hide Chat in Images",
         description = "Whether to hide the chat box and private messages when capturing screenshots.<br/>" +
-            "Note: visually you may notice the chat box momentarily flicker as it is hidden for the screenshot.",
+            "Note: visually you may notice the chat box momentarily flicker as it is hidden for the screenshot.<br/>" +
+            "Warning: 'Hide Split PMs' has no effect if 'Split friends private chat' is not enabled in the game settings",
         position = 1010,
         section = advancedSection
     )
-    default boolean screenshotHideChat() {
-        return false;
+    default ChatPrivacyMode chatPrivacy() {
+        return ChatPrivacyMode.HIDE_SPLIT_PM;
```

Comment:
do we want a new default that requires an in-game setting? i know in theory hide pms is probably the most useful, but old default is off, as i'm sure lots of people don't have unhinged friends

Reply:
note: for ppl on the old default, they get migrated to hide none in the new enum

in general i think most users would want to hide dms but yeah the default value having an associated warning isn't ideal. technically we could detect whether the game setting is correct & warn in chat on login?

Reply:
i changed my mind, i don't know why i previously took the approach of osrs players _not_ having unhinged friends, this is a better default